src/main/java/it/pagopa/pn/delivery/rest/PnInternalNotificationsController.java

```diff
@@ -212,6 +213,9 @@ public ResponseEntity<NotificationAttachmentDownloadMetadataResponse> getReceive
                     docIdx,
                     false
             );
+            @NotNull String filename = response.getFilename();
+            @NotNull String safeUrl = response.getUrl().split("\\?")[0];
```

Comment:
Possibile che getUrl sia null o non contenga ?

Reply:
ho visto che la getUrl potrebbe essere null poiché se il file non è ancora disponibile dovrebbe essere valorizzato il campo retryAfter e non l'url. In caso non sia presente '?' la regexp fallisce e viene restituito l'url intero

Reply:
Allora metti anche il retryAfter nel log, quando è disponibile

Reply:
la stessa modifica deve essere riportata sull'apertura dei bollettini di pagamento e negli atti opponibili

Reply:
per i bollettini di pagamento posso farlo nella stessa issue. Per gli atti opponibili devo aprire issue dedicata